test/fixtures/test/states.json

```diff
@@ -39,8 +39,19 @@
     "status": "done",
     "options": {
       "iconType": "far",
-      "icon": "minus-octagon",
+      "icon": "octagon-minus",
       "color": "orange"
     }
+  },
+  {
+    "id": "77777",
+    "name": "THMG Transfered",
+    "sequence": 4,
+    "status": "done",
+    "options": {
+      "iconType": "fas",
+      "icon": "arrow-right-arrow-left",
```

Comment:
@nmajor25 I added this in the tests as this icon is used on THMG and maybe elsewhere but not in our code.  I almost removed it from the build because of this.  This should prevent accidentally pulling the icon